components/Request/Request.jsx

```diff
@@ -8,6 +8,7 @@ import { REDUCER_KEY } from '../../reducers';
 
 const Request = ({ error, matchId, loading, progress, dispatchRequest, dispatchMatchId }) => {
   function submit() {
+    document.getElementById('match_id').value = '';
```

Comment:
Forgive my ignorance of redux and reducers right now, but do you mean I do something like this:
`case requestActions.MATCH_ID:
      return {
        ...state,
        matchId:` **action.matchId**`,
      };`

###### in the reducer and this in Request.jsx:

`dispatchRequest(matchId);`
 `dispatchMatchId('');`

***
Because that does indeed work in changing the state. I can see why it would not be linked up for testing purposes.